keyboards/handwired/wwa/rosetta/readme.md

```diff
@@ -0,0 +1,27 @@
+# rosetta
+
+![rosetta](https://i.imgur.com/KQL8cUe.jpg)
```

Comment:
```suggestion
![rosetta](https://i.imgur.com/KQL8cUeh.jpg)
```

Reply:
I don't understand this change, the URL you suggested directs to a 404